Solutions/Menes.PetStore/Menes.PetStore.csproj

```diff
@@ -20,7 +20,7 @@
       <PrivateAssets>all</PrivateAssets>
       <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
     </PackageReference>
-    <PackageReference Include="Microsoft.Extensions.Http" Version="3.1.19" />
+    <PackageReference Include="Microsoft.Extensions.Http" Version="[3.1.*,)" />
```

Comment:
What's going on with the square brackets/round brackets here?

Reply:
That's what a floating version looks like.

Square brackets denote an inclusive bound, round ones an exclusive bound. So if you wanted to exclude v5 or later you could write `[3.1.*,5)`. If for some reason there was a very narrow range of versions that you know work, you could write `[3.1.12, 3.1.18]` to say it works with versions 3.1.12 up to and including 3.1.18 but nothing later than that. Leaving an empty value for the upper bound indicates that we impose no upper bound.

And the `*` enables floating. In this case, it says the lower bound floats. This says our lower bound is whatever the latest `3.1.something` version is (`3.1.20` right now) (but the `packages.lock.json` file takes precedence, so in CI build environments where things are configured not to update that file during restore, this lower bound gets overruled by the lock file).